sklearn/tests/test_pipeline.py

```diff
@@ -1072,3 +1072,13 @@ def test_make_pipeline_memory():
     assert len(pipeline) == 2
 
     shutil.rmtree(cachedir)
+
+
+def test_pipeline_param_error():
+    clf = make_pipeline(LogisticRegression())
+    assert_raise_message(TypeError,
```

Comment:
please use `pytest.raises` instead:

```py
with pytest.raises(TypeError, match=msg):
    clf.fit(...)
```

Reply:
Should I test on the whole message or just a keyword? Current I'm checking on 'sample_weight' for the exception message.